frontend/packages/kubevirt-plugin/src/utils/guest-agent-strings.ts

```diff
@@ -1,31 +1,29 @@
-import {
-  GUEST_AGENT_REQUIRED_MESSAGE,
-  NOT_AVAILABLE_MESSAGE,
-  NO_LOGGED_IN_USERS_MSG,
-  VM_NOT_RUNNING_MESSAGE,
-} from '../strings/vm/messages';
+import { TFunction } from 'i18next';
 import { pluralize } from './strings';
 import { VMStatus } from '../constants/vm/vm-status';
 
-export const getNumLoggedInUsersMessage = (numLoggedInUsers: number | null) => {
+export const getNumLoggedInUsersMessage = (t: TFunction, numLoggedInUsers: number | null) => {
   if (numLoggedInUsers == null) {
-    return NOT_AVAILABLE_MESSAGE;
+    return t('kubevirt-plugin~Not available');
   }
 
   if (numLoggedInUsers === 0) {
-    return NO_LOGGED_IN_USERS_MSG;
+    return t('kubevirt-plugin~No users logged in');
   }
 
   return `${numLoggedInUsers} ${pluralize(numLoggedInUsers, 'user')}`;
```

Comment:
Shouldn't this be translated using `count`?

Reply:
Yeee, i18n is great, using `count` now